examples/demo_all.pp

```diff
@@ -31,6 +31,7 @@
   include ciscopuppet::demo_command_config
   include ciscopuppet::demo_domain
   include ciscopuppet::demo_interface
+  #include ciscopuppet::demo_interface_service
```

Comment:
OK, I had thought the plan at this point is to use facter to constrain the demo elements to the platforms that support them.